extensions/wikia/PageHeader/Subtitle.class.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace Wikia\PageHeader;
+
+use FakeSkin;
+use Html;
+use RequestContext;
+
+class Subtitle {
+	/**
+	 * @var string
+	 */
+	public $subtitle;
+
+	/**
+	 * @var string
+	 */
+	public $pageSubtitle;
+
+	/**
+	 * @var mixed|null
+	 */
+	public $suppressPageSubtitle;
+
+	/**
+	 * @var \SkinTemplate
+	 */
+	private $skinTemplate;
+
+	/**
+	 * @var bool
+	 */
+	private static $onEditPage = false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$onEditPage" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)